.github/workflows/doc.yml

```diff
@@ -12,3 +12,33 @@ jobs:
       # graphviz is required to build mermaid graph
       # optipng is required to optimize thumbnail
       install_package_ubuntu: graphviz optipng
+
+  Push_dev:
+    needs: Build
+    # Push only on the "RELEASE_next_minor" and update the "dev" documentation
```

Comment:
I am not sure if I fully understand your question. The logic is as follow:
- ["current" doc](https://github.com/hyperspy/hyperspy-doc/tree/gh-pages/current) contains the latest release (patch, minor, major) and this is why "on tag", it should be pushed to that folder
- ["dev" doc](https://github.com/hyperspy/hyperspy-doc/tree/gh-pages/dev) contains the documentation of the `RELEASE_next_minor` branch. `RELEASE_next_patch` is merged regularly into  `RELEASE_next_minor` and the corresponding release note of that branch will then appear there.

An alternative would be add a third documentation build for the `RELEASE_next_patch` branch, then we will have:
- current
- dev_minor
- dev_patch

but I am not sure if this will be very obvious for users the difference between "dev_minor" and "dev_patch".

Reply:
I think I got mixed up, because at the moment, the selector offers `dev` and then `2.0`, but of course the latter is the `current` doc and would be replaced by the patch release once out. So everything should be fine